### PR TITLE
Fix mobile context menu

### PR DESCRIPTION
**Pull request by rudolfs**
Refactor the `MobileNavbar` component and move it into `Header`. This fixes a bug where it wasn't possible to close the context menu once it was open.

https://user-images.githubusercontent.com/158411/191220588-e0c891e6-6b35-44f4-9799-15dd9174c53d.mov

### CODE REVIEW

File: src/Header.svelte [hunk 1]
```diff
@@ -12,13 +12,13 @@
...
   import Search from "@app/Search.svelte";
   import Floating from "@app/Floating.svelte";
-  import Icon from "./Icon.svelte";
-  import MobileNavbar from "./MobileNavbar.svelte";
   import SeedDropdown from "./SeedDropdown.svelte";
   import ThemeToggle from "./ThemeToggle.svelte";
   import Button from "@app/Button.svelte";
   import SearchResults from "@app/components/Modal/SearchResults.svelte";
   import type { ResolvedSearch } from "@app/resolver";
+  import Icon from "./Icon.svelte";
```
Comment (sebastinez) [inline src/Header.svelte:20]:
```suggestion
  import Icon from "@app/Icon.svelte";
```

File: src/Header.svelte [hunk 2]
```diff
@@ -12,13 +12,13 @@
   import Avatar from "@app/Avatar.svelte";
   import Search from "@app/Search.svelte";
   import Floating from "@app/Floating.svelte";
-  import Icon from "./Icon.svelte";
-  import MobileNavbar from "./MobileNavbar.svelte";
   import SeedDropdown from "./SeedDropdown.svelte";
   import ThemeToggle from "./ThemeToggle.svelte";
```
Comment (sebastinez) [inline src/Header.svelte:16]:
```suggestion
  import SeedDropdown from "@app/SeedDropdown.svelte";
  import ThemeToggle from "@app/ThemeToggle.svelte";
```

**Review by sebastinez — APPROVED** [review]
Nice!
Just saw that we weren't importing some components from `@app` but from their relative path. Maybe we can fix that.

**Comment by rudolfs** [comment]
> Just saw that we weren't importing some components from `@app` but from their relative path. Maybe we can fix that.

Yeah, force pushed a fixup, also sorted the imports.

**Review by cloudhead — APPROVED** [review]

**Pull request merged**